tests/deepl_api/deepl_api_test.py

```diff
@@ -1048,14 +1048,20 @@ def test_deepl_bulk_mt_no_target_language(
         )
 
 
+# Possible errors from DeepL API
+api_errors = [404, 413, 429, 456, 500]
+
+
 @pytest.mark.django_db
 @pytest.mark.parametrize(
     "login_role_user", PRIV_STAFF_ROLES + [AUTHOR, MANAGEMENT, EDITOR], indirect=True
 )
-@pytest.mark.skip(reason="server error handling is currently not implemented")
+@pytest.mark.parametrize("api_errors", api_errors)
+# pylint: disable=redefined-outer-name
 def test_deepl_bulk_mt_api_error(
     load_test_data: None,
     login_role_user: tuple[Client, str],
+    api_errors: int,
```

Comment:
```suggestion
    error: int,
```
A bit nitpicky, but maybe you could rename this to `error`, since right now it sounds like it could store multiple errors